osu.Game.Rulesets.Osu/Mods/OsuModTransform.cs

```diff
@@ -40,27 +81,61 @@ private void applyTransform(DrawableHitObject drawable, ArmedState state)
                     return;
 
                 default:
-                    var hitObject = (OsuHitObject)drawable.HitObject;
-
-                    float appearDistance = (float)(hitObject.TimePreempt - hitObject.TimeFadeIn) / 2;
+                    var h = (OsuHitObject)drawable.HitObject;
```

Comment:
Why was this variable renamed?

Reply:
fixed. iirc i changed it because i saw other placed abbreviating as h